src/Pages/Community/CommunityHome/Components/FilterTapCard.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./FilterTapCard.scss";
+
+class FilterTapCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    const { id, name, subTap } = this.props.filterTapCard;
+    return (
+      <>
+        <div className="FilterCard">
+          <div className="filter addCursor" key={id}>
+            {`${name}`}
```

Comment:
다른 스트링과 같이 쓰지 않는다면 이렇게 하면 될 것 같네요!
```suggestion
            {name}
```